content/faq/libraries-frameworks/data.mdx

```diff
@@ -19,6 +19,7 @@ showToc: true
 - [LINQ to DB](https://github.com/linq2db/linq2db) is the fastest LINQ database access library offering a simple, light, fast, and type-safe layer between your POCO objects and your database.
 - [Dapper](https://github.com/StackExchange/Dapper)
 - [NHibernate](https://github.com/nhibernate/nhibernate-core) NHibernate is a mature, open source object-relational mapper for the .NET framework.
+- [dotConnect](https://www.devart.com/dotconnect/) (not free) can connect to Oracle,MySQL,PostgreSQL,SQLite,DB2 and SQL Server. Quicky connect to database without install any ODBC drivers
```

Comment:
This is good, just very minor comment.
Please add a space after comma to separate each database system.

Please also add other benefit of using dotConnect. I think NHiberante and EF supports all database in that list and does not need any additional ODBC drivers as well because they use ADO.NET under the hood.

Reply:
in case of Oracle usage, Dapper requied Oracle.ManagedDataAccess.Core but dotConnect don't need any nested dll. 


ref: https://www.devart.com/dotconnect/oracle/docs/ComparingProviders.html
ref: https://stackoverflow.com/questions/43054955/net-4-6-oracle-manageddataaccess-requirements
ref: https://www.c-sharpcorner.com/article/net-core-with-oracle-database-using-dapper/


in case of DB2 usage, Dapper need ODBC too.
ref: https://jrcook.dev/2016/10/24/using-dapper-with-db2/


btw I dont have experienced with NHibernate krub.